chapter_09_docker.asciidoc

```diff
@@ -58,8 +58,6 @@ Let me know how you got on, if you get stuck on anything,
 if any explanations don't make sense,
 or if any of the instructions don't work for you.
 
-I hope you enjoy the new version!
```

Comment:
I'm not sure why this line shows as deleted. I just fetched and rebased from upstream. I'll have another look soon/

Reply:
I must have had it open in vim while I fetched and rebased but then didn't reload the file, so I just saved it without those lines. Fixed now